h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -195,10 +210,17 @@ export default function CreateEditGroupForm() {
     let response: CreateUpdateGroupAPIResponse;
 
     try {
+      const body: CreateUpdateGroupAPIRequest = {
```

Comment:
This ensures that the request body is compatible with the schema specified in `api.ts` (if not, TS will complain).